pkg/utils/ufs_path_builder.go

```diff
@@ -41,6 +41,22 @@ func (u UFSPathBuilder) GenAlluxioMountPath(curMount datav1alpha1.Mount, mounts
 	return fmt.Sprintf(common.AlluxioMountPathFormat, curMount.Name)
 }
 
+// dataset.spec.mounts mount to goosefs instance strategy:
```

Comment:
I have concern on this package. Seems it's the utils for alluxio or Goose instead of all the runtime. If we really think it's not common. Maybe the common part can be placed in UFSPathBuilder. How about provide AlluxioUFSPathBuilder and GooseUFSPathBuilder?


```golang
type GooseUFSPathBuilder struct{
*UFSPathBuilder
}
```

Reply:
LGTM， @allenhaozi WDYT

Reply:
I want to know the difference, right?
Whether they can all follow one rule ?

Reply:
At present, it seems that it can be consistent, but we want to maintain expansibility and isolation.

Reply:
Is it fine to be consistent now? We can support `GooseUFSPathBuilder ` in future.

Reply:
I will seperate it in the future. @cheyang